ui/components/app/modals/deposit-token-modal/index.scss

```diff
@@ -0,0 +1,118 @@
+.deposit-token-modal {
```

Comment:
Many of the styles in this file can be avoided by using our design system components, Box and Typography. Please attempt to use those for new components.